CHANGELOG.md

```diff
@@ -6,10 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [13.0.0]
+### Changed
+- **BREAKING:** Updated `SmartTransactionsController` to inherit from `StaticIntervalPollingController` instead of `StaticIntervalPollingControllerV1` ([#397](https://github.com/MetaMask/smart-transactions-controller/pull/397)).
+  - The constructor for `SmartTransactionsController` now accepts a single options object instead of three separate arguments, with configuration options merged into this object.
+  - `SmartTransactionsController` now requires a `messenger` option (with the corresponding type `SmartTransactionsControllerMessenger` now available).
+  - The constructor no longer accepts `onNetworkStateChange`; instead, it subscribes to `NetworkController:stateChange`.
+  - The `getNetworkClientById` argument has been removed from the constructor and is now accessed through the messenger.
+  - The controller no longer subscribes to its own events; this is now managed via the messenger.
+  - Event emission is no longer handled by `EventEmitter`; the messenger is now used for emitting events.
+  - The `SmartTransactionsControllerConfig` type has been removed and replaced with `SmartTransactionsControllerOptions`.
+  - Added and exported the following types: `SmartTransactionsControllerMessenger`, `SmartTransactionsControllerState`, `SmartTransactionsControllerGetStateAction`, `SmartTransactionsControllerActions`, `SmartTransactionsControllerStateChangeEvent`, `SmartTransactionsControllerSmartTransactionEvent`, and `SmartTransactionsControllerEvents`.
+
 ## [12.0.1]
+### Removed
+- Remove logic for ensuring uniqueness of smart transactions ([#404](https://github.com/MetaMask/smart-transactions-controller/pull/404))
```

Comment:
this change is related to this discussion https://github.com/MetaMask/smart-transactions-controller/pull/407#discussion_r1727629351

Reply:
We usually try to keep RCs separate from other changes -  do you think we can address this in another PR?

Reply:
Thanks Michele 12.0.1 changelog updates extracted to a separate PR